skimage/segmentation/tests/test_clear_border.py

```diff
@@ -24,9 +24,24 @@ def test_clear_border():
     assert_array_equal(result, np.zeros(result.shape))
 
     # test background value
-    result = clear_border(image.copy(), 1, 2)
+    result = clear_border(image.copy(), buffer_size=1, bgval=2)
     assert_array_equal(result, 2 * np.ones_like(image))
 
 
+def test_clear_border_non_binary():
+    image = np.array([[1, 2, 3, 1, 2],
+                      [3, 4, 5, 4, 2],
+                      [3, 4, 5, 4, 2],
+                      [3, 3, 2, 1, 2]])
```

Comment:
In this image, just setting all the borders to 0 would work (without regard to labels). I would replace one of the central pixels with `1` to make it a more robust test.